msrezaie/city-scrapers-wichita
# 🏗️ Build spider: Sedgwick Juvenile Corrections Board

## What's this PR do?
Adds a spider to scrape meetings from the website of Sedgwick county Juvenile Corrections Advisory Board, Wichita KS. The new spider is called wicks_sedgwick_jcab.

## Why are we doing this?
Built the spider as per request.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl wicks_sedgwick_jcab -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid.

## Are there any smells or added technical debt to note?
In the organization's website the details for each entry of the meeting is provided in a `<li>` tag as part of a single line which includes the meeting date, agenda & minutes links, and for some entries there's either a status string set to 'CANCELLED' or a different address for the meeting's location. Parsing of this location was not done as there was only one instance of it appearing among the meeting entries.
